hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/AbstractDsAssignmentStrategy.java

```diff
@@ -196,9 +201,19 @@ protected List<Long> closeObsoleteUpdateActions(final Collection<Long> targetsId
      * @param actionId
      *            the action id of the assignment
      */
-    void cancelAssignDistributionSetEvent(final Target target, final Long actionId) {
+    void cancelAssignDistributionSetEvent(final Action action) {
```

Comment:
please check visibility

Reply:
Defined visibility to all assignment strategy related classes